semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/MemberOps.scala

```diff
@@ -0,0 +1,42 @@
+package scala.meta.internal.semanticdb.scalac
+
+import scala.{meta => m}
+
+trait MemberOps { self: DatabaseOps =>
+  private[this] lazy val ignoreName: Set[g.Name] = Set(
+    g.nme.CONSTRUCTOR,
+    g.nme.MIXIN_CONSTRUCTOR,
+    g.nme.asInstanceOf_,
+    g.nme.asInstanceOf_Ob,
+    g.nme.isInstanceOf_,
+    g.nme.isInstanceOf_Ob,
+    g.nme.hashCode_,
+    g.nme.HASHHASH,
+    g.nme.ne,
+    g.nme.eq,
+    g.nme.finalize_,
+    g.nme.wait_,
+    g.nme.EQ,
+    g.nme.NE,
+    g.nme.synchronized_,
+    g.nme.notify_,
+    g.nme.notifyAll_,
+    g.nme.clone_,
+    g.nme.equals_,
+    g.nme.toString_,
+    g.nme.getClass_
+  )
+
+  implicit class XtensionGTypeMSignatures(tpe: g.Type) {
+    def lookupMembers: List[m.Signature] = {
```

Comment:
Looking forward to remove this hack!

Reply:
Can we add `@deprecated` to remind us to remove this someday?

Reply:
Done.

Reply:
The build fails now, because we have fatal warnings. I'll open a ticket.

Reply:
https://github.com/scalameta/scalameta/issues/1261